### PR TITLE
[Hjson] Add v4.4.0

### PR DESCRIPTION
**Pull request by Wikunia**
Couldn't test this locally as I have some problems running go in BinaryBuilder on MacOS so hope to get some information on the runs here if something fails.

### CODE REVIEW

**Comment by giordano** [comment]
> I have some problems running go in BinaryBuilder on MacOS

Many people are reporting issues lately, apparently it was a breaking change in Docker: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/issues/389

File: H/Hjson/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+products = [
+    ExecutableProduct("hjson-cli", :hjson),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; compilers=[:c, :go], julia_compat="1.6")
```
Comment (giordano) [inline H/Hjson/build_tarballs.jl:34]:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; compilers=[:c, :go], julia_compat="1.6")

```

File: H/Hjson/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,34 @@
+using BinaryBuilder
+
+name = "Hjson"
+
+hash = "bdaa45f5d9768665e3081b9fe64cd574f9e092d4"
+version = v"4.4.0"
+
+# Collection of sources required to build pprof
+sources = [
+    GitSource("https://github.com/hjson/hjson-go", hash),
```
Comment (giordano) [inline H/Hjson/build_tarballs.jl:10]:
```suggestion
    GitSource("https://github.com/hjson/hjson-go.git", hash),
```

**Comment by Wikunia** [comment]
I guess I have to build the main.go?

**Comment by giordano** [comment]
The error message is
```

**Comment by Wikunia** [comment]
I'll have a look the next days. Thanks @giordano

File: H/Hjson/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+version = v"4.4.0"
+
+# Collection of sources required to build pprof
+sources = [
+    GitSource("https://github.com/hjson/hjson-go.git", hash),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/hjson-go/
```
Comment (giordano) [inline H/Hjson/build_tarballs.jl:15]: This may work:
```suggestion
cd $WORKSPACE/srcdir/hjson-go/hjson-cli
```

**Comment by giordano** [comment]
Thanks for your contribution!

**Comment by Wikunia** [comment]
Thanks for the help and quick merge @giordano

**Pull request merged**